bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -498,7 +498,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- Default surface roughness parameterization                         -->
 <!-- ================================================================== -->
 
-<z0param_method>ZengWang2007</z0param_method>
+<z0param_method phys="clm4_5">ZengWang2007</z0param_method>
+<z0param_method phys="clm5_0">ZengWang2007</z0param_method>
+<z0param_method phys="clm5_1">Meier2022</z0param_method>
 
 <use_z0m_snowmelt z0param_method="Meier2022" >.true.</use_z0m_snowmelt>
 <use_z0m_snowmelt >.false.</use_z0m_snowmelt>
```

Comment:
I just looked at the CESM3_dev branch to see if there's anything else that should be done, and you should add this part:

``` XML 

 <use_z0m_snowmelt z0param_method="Meier2022" >.true.</use_z0m_snowmelt>
-<use_z0m_snowmelt >.false.</use_z0m_snowmelt>
+<use_z0m_snowmelt                            >.false.</use_z0m_snowmelt>

```

Reply:
@ekluzek I think I see these two lines already in place (lines 505-506 above).
Unless I am misunderstanding what you mean.

Reply:
No, you are correct. It was already in place. It must not have been on place on the CESM3_dev, so looking at it made me think we needed to add it in again.

So resolving.